viewer/src/views/SplashScreen.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div :class="$style.splashscreen" class="scrollbar">
+    <div :style="config.style" :class="$style.markdown" class="flex-grow-1" v-html="splashScreenMd" />
+    <b-button size="is-large" :label="buttonValidateLabel" :class="$style.buttonOkay" @click="validation" />
+  </div>
+</template>
+
+<script lang="ts">
+import { SplashScreenConfig } from "@/@types/splashscreen";
+import marked from "marked";
+import { TranslateResult } from "vue-i18n";
+import { Component, Emit, Vue } from "vue-property-decorator";
+
+@Component
+export default class SplashScreen extends Vue {
+  get config(): SplashScreenConfig {
+    return this.$uiStore.splashScreenConfig!;
+  }
+
+  get splashScreenMd(): string {
+    return marked(this.$uiStore.splashScreenData!);
+  }
+
+  get buttonValidateLabel(): TranslateResult {
+    return this.config.buttonValidateLabel ?? this.$t("splashScreen.button.validation");
+  }
+
+  @Emit()
+  validation() {}
+}
+</script>
+
+<style lang="scss" module>
+.splashscreen {
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  padding: 32px;
+}
+.buttonOkay {
+  min-width: 310px;
+  align-self: center;
+}
```

Comment:
That default minimum width is arbitrary and not necessary.
It makes the button ridiculously wide with the default label value.
Not everyone is a fan of Windows 8's style.

There is also some missing padding/margin below the button when the notice's text is long enough to cause the box to scroll:

![16253343481887](https://user-images.githubusercontent.com/1449319/124362769-48f10880-dc37-11eb-8ded-bb4eb9f55cce.png)



The name of the class should also match the name of the parameter.

```suggestion
.buttonAcknowledge {
  align-self: center;
}
```

Reply:
> That default minimum width is arbitrary and not necessary.

The button, by default, is too small and not visible enough.
310px isn't arbitrary, it slightly smaller than a mobile screen (320px), so it looks pretty good on mobile.

> There is also some missing padding/margin below the button when the notice's text is long enough to cause the box to scroll.

Weird, I have padding all around the SplashScreen. You'll have to show me your screen.

Okay for the button's name.

Reply:
> > There is also some missing padding/margin below the button when the notice's text is long enough to cause the box to scroll.
> 
> Weird, I have padding all around the SplashScreen. You'll have to show me your screen.

You can see it in the screenshot in the previous message you replied to. You can reproduce it by putting a long document in "splashscreen.md" to cause an overflow. Scrolling at the bottom, you'll see that this padding/margin is oddly not affecting the bottom of the button. I have no idea of why that is the case.

Reply:
This is a bug in FireFox. Let's FFox it.

Reply:
Bug observed with Firefox 78.11.0esr, but gone on 89.0.2.
Since it's not a critical issue impeding the use of the software, we can just ignore it.